playwright/NOTES.txt

```diff
@@ -0,0 +1,8 @@
+
+npm run cucumber -- features/homePageElements.feature 
+will only work if you also comment out the paths: ["playwright/features/**/*.feature"], line in the cucumber.js file. It seems that cucmber.js paths option is also being sent whenever you pass the specific feature file argument, so it’s running all the tests and the one you’re passing in as an argument on the command line.
+
+actually need to run npm run cucumber -- playwright/features/homePageElements.feature to specify the playwright directory at the root
```

Comment:
also, thank you for adding this. let's actually just move it to the main README.md file so that we keep these things in one place. we can add a section to the README called ## Playwright or something.

Reply:
NOTES.txt is something I add to my projects for personal use, so this can just be removed from this repo.  Nice idea to add some of the notes to the Readme file.